src/test/kotlin/io/github/rybalkinsd/kohttp/ext/RequestExtTest.kt

```diff
@@ -0,0 +1,27 @@
+package io.github.rybalkinsd.kohttp.ext
+
+import okhttp3.Request
+import org.junit.Test
+import kotlin.test.assertEquals
+
+/**
+ * @author doyaaaaaken
+ */
+class RequestExtTest {
```

Comment:
I remember that you will write more tests after our approval, but small comment - you should focus your attention on headers and also cookies